### PR TITLE
fix: heat map axes affected by local time

### PR DESCRIPTION
**Pull request by gikf**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/47073

---

- Depending on the local time axes can start on the different value (see occurrences in the linked issue). `timeFormat` method interprets data as being in the local time, `utcFormat` uses _Coordinated Universal Time (UTC)_ for that.
- The third occurrence `timeFormat('%Y - %B')` did not seem to impact tests, but I've changed it for consistency.
- Example with applied changes can be found at https://codepen.io/sanityto/full/gOKJQae

### CODE REVIEW

**Review by ShaunSHamilton — APPROVED** [review]
I am happy with this 👍 

Thank you, @gikf

**Review by naomi-lgbt — APPROVED** [review]

**Comment by gikf** [comment]
Is the deployment automatic, or it might need some manual actions? I'm asking, because old code still appears at the https://heat-map.freecodecamp.rocks

**Comment by ShaunSHamilton** [comment]
@gikf Thanks for the reminder. No, deployments are not automatic.

However, we should double-check this is not blocked by the current updates @raisedadead is doing?

**Comment by raisedadead** [comment]
Hey, @scissorsneedfoodtoo or @moT01 do you mind doing one last deployment to the demo-projects before we freeze further PR merges?

**Comment by naomi-lgbt** [comment]
@raisedadead I went ahead and deployed.

**Comment by raisedadead** [comment]
Thanks @naomi-lgbt

**Comment by gikf** [comment]
Thanks everybody!

**Pull request merged**
